webapp/channels/src/components/threading/global_threads/global_threads.tsx

```diff
@@ -228,7 +228,7 @@ const GlobalThreads = () => {
                                     >
                                         {chunks}
                                     </Link>
-                                ),
+                                ) as ReactNode,
```

Comment:
Is this pattern correct? I think we might be telling TS that `(chunks) => {}` is a `ReactNode`, not that it's returning a ReactNode

Reply:
Looks like the type signature changed and now has a constraint as of https://github.com/formatjs/formatjs/commit/0d03bb66123cb49fbd1c7d27908979bc4521b41f. (For now at least, perhaps until it's no longer required by react-intl,) I think it'd be better to explicitly pass it as the type parameter (`formatMessage<ReactNode>`), rather than coerce/cast each individual value return type.

Reply:
that sounds good, changed to type passing